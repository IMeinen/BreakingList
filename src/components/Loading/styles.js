import styled from 'styled-components';

import ReactLoading from "react-loading";

export const CustomLoader = styled(ReactLoading)`
  margin: auto;
  height: 120px;
  width: 120px;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);

  color: #fff;
`;
