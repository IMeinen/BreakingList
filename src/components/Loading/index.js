import React from 'react';

import { CustomLoader } from './styles';

export default function Loading() {
  return (
    <CustomLoader type="bars"  />
  );
}
