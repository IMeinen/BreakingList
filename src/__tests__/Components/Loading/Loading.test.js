import React from 'react';

import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Loading from '../../../components/Loading';

Enzyme.configure({ adapter: new Adapter() });
describe('Loading component tests', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
