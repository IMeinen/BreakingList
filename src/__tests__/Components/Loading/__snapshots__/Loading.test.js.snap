// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component tests renders correctly 1`] = `
<Styled(Loading)
  type="bars"
/>
`;
